modules/apps/site/site-api/src/main/java/com/liferay/site/display/context/GroupDisplayContextHelper.java

```diff
@@ -25,6 +31,16 @@ public GroupDisplayContextHelper(HttpServletRequest httpServletRequest) {
 		_httpServletRequest = httpServletRequest;
 	}
 
+	public List<BreadcrumbEntry> getBreadcrumbEntries() {
+		return Collections.singletonList(
+			new BreadcrumbEntry() {
+				{
+					setTitle(
+						LanguageUtil.get(LocaleUtil.getDefault(), "staging"));
+				}
+			});
+	}
+
```

Comment:
Inlining does not necessarily mean to put everything in a single expression. It is fine to have multiple statements; can you refactor this into something like:
```
BreadcrumbEntry breadcrumbEntry = ...;

breadcrumbEntry.setTitle(...);

return Collections.singletonList(breadcrumbEntry);
```